api/api-rules/violation_exceptions.list

```diff
@@ -206,7 +206,11 @@ API rule violation: list_type_missing,k8s.io/api/extensions/v1beta1,RunAsUserStr
 API rule violation: list_type_missing,k8s.io/api/extensions/v1beta1,RuntimeClassStrategyOptions,AllowedRuntimeClassNames
 API rule violation: list_type_missing,k8s.io/api/extensions/v1beta1,SupplementalGroupsStrategyOptions,Ranges
 API rule violation: list_type_missing,k8s.io/api/imagepolicy/v1alpha1,ImageReviewSpec,Containers
+API rule violation: list_type_missing,k8s.io/api/networking/v1,HTTPIngressRuleValue,Paths
 API rule violation: list_type_missing,k8s.io/api/networking/v1,IPBlock,Except
+API rule violation: list_type_missing,k8s.io/api/networking/v1,IngressSpec,Rules
+API rule violation: list_type_missing,k8s.io/api/networking/v1,IngressSpec,TLS
+API rule violation: list_type_missing,k8s.io/api/networking/v1,IngressTLS,Hosts
```

Comment:
https://kubernetes.io/docs/reference/using-api/api-concepts/#merge-strategy

Rather than adding the violations here, you should probably annotate these fields 
with `//+listType=atomic` which is:
1. Probably not always what you wanted in the first place, but right now it's not a backward compatible change
2. More explicit than relying on the default (which requires the exception).